### ISSUE TITLE
Oplysning om hvornår fikspunkt senest har været brugt som fastholdt.

### ISSUE THREAD

**Issue by Palle-N** [Kortforsyningen/FIRE#568]
Tidligere var det sådan at man kunne se hvornår et fikspunkt senest har været brugt som fastholdt i en beregning, det er ikke muligt nu. Det er en go´ oplysning at have når man i en opgave skal vurdere hvilke punkter man skal tilslutte en indmåling af et nyetableret punkt.

Er det en mulighed at tilføje den oplysning ved udtræk af oplysninger?

124-09-09101   6257662   494355     20.3850       2002/1985  2001 03 22, 09.46 
124-09-09102   6258312   495939     15.8680                2003  2003 06 11, 15.17 
124-09-09103   6258464   496907     13.6130                2003  2003 06 11, 15.17 
124-09-09104   6258493   497748     10.2490                2003  2003 06 11, 15.17 
124-09-09105   6258544   498160     11.7240                2003  2003 06 11, 15.17 
124-09-09106   6260015   498162     10.7790       2014/1985  2001 03 22, 09.46 
124-09-09109   6254947   497722     21.2270       2002/1985  2001 03 22, 09.46 
124-09-09110   6253523   496313     22.7680       2002/1985  2001 03 22, 09.46 
124-09-09112   6257301   497846     16.3150                1985  2001 03 22, 09.46

**Comment by kbevers** [comment]
Nej, det er desværre ikke muligt. Den information er ikke lagret i databasen. Hvordan fandt du ud af det førhen?

**Comment by Palle-N** [comment]
Jeg mener det automatisk blev noteret i udtrækket, du kan se et udtræk af højdefikspunkter her F:\GRF\Data\GEO\LM\Fixopl2020\JYLLAND
Filen hedder "koterjyl"

Hvis der bare står eks. 1985 er koten fra 1985, står der eks. 2003/1985 er koten fra 1985, og fastholdt i en beregning fra 2003.

**Comment by kbevers** [comment]
Hvordan er udtrækket lavet i sin tid? Altså, hvilket program er kørt osv. Ved du det?

**Comment by nho-sdfe** [comment]
Det var et helt standard output fra refgeo udtrukket med fromcrd.

**Comment by kbevers** [comment]
Okay. Jeg kan se at man i REFGEO har haft registreret to årstal på koordinater, årstallet for seneste opdatering og årstallet for seneste fastholdelse. Det har vi ikke ført videre i FIRE så det er desværre ikke muligt at gøre fremadrettet.

**Comment by kbevers** [comment]
Attributten `ATTR:muligt_datumstabil` er egentligt tiltænkt samme formål, men den er jo ikke sat på punkter der ikke har været under revision før 2021. Men pointen med den er blandt andet at give en ide om hvad der kan fastholdes i beregninger. Jeg vil prøve at undersøge om den kan retrofittes ud fra information i REFGEO.

**Comment by kristiankeller** [comment]
Men beregningen findes et sted, og så må man kunne se efter der? JA, det er mere besværligt, men det må kkunne fungere. Jeg har faktisk samme problem da jeg skal se hvilke punkter der er fastholdt ved KDI-niv-beregningen gennem de sidste 10 år... De gamle beregninger antager jeg ligger under /REGN/ eller beregnerens initialer på REFFS1t ?
